routes/index.js

```diff
@@ -97,6 +114,69 @@ router.get('/room/:name', function (req, res) {
   }
 });
 
+/**
+ * POST /captions/start
+ */
+router.post('/captions/start', async function (req, res) {
+  // With custom expiry (Default 30 days)
+  const expires = Math.floor(new Date() / 1000) + (24 * 60 * 60);
+  const projectJWT = projectToken(apiKey, secret, expires);
+  const captionURL = `${captionsUrl}/${apiKey}/captions`;
+
+  const captionPostBody = {
+    sessionId: req.body.sessionId,
+    token: req.body.token,
+    languageCode: 'en-US',
+    maxDuration: 36000,
+    partialCaptions: 'true',
+  };
+
+  try {
+    captionResponse = await axios.post(captionURL, captionPostBody, {
+      headers: {
+        'X-OPENTOK-AUTH': projectJWT,
+        'Content-Type': 'application/json',
+      },
+    });
+  } catch (err) {
+    console.warn(err);
+    res.status(500);
+    res.send(`Error starting transcription services: ${err}`);
+    return;
+  }
+
+  res.send(captionResponse.data.captionsId);
+});
+
+/**
+ * POST /captions/stop
+ */
+router.post('/captions/stop', postBodyParser, async function (req, res) {
+  const captionsId = req.body.captionId;
```

Comment:
Nit, `captionId` vs `captionsId` 😅

Reply:
I already have a higher-scope `captionId`, don't want to confuse people with that one 😓